sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -39,9 +39,16 @@ class PlanParserSuite extends AnalysisTest {
     comparePlans(parsePlan(sqlCommand), plan, checkAnalysis = false)
   }
 
+  private def assertQueryEqual(query: String, plan: LogicalPlan): Unit = {
```

Comment:
This makes it hard to write tests as people need to pick between `assertEqual` and `assertQueryEqual`.

How much we can speed up  this test suite by doing so?

Reply:
About tens of milliseconds